packages/core/helper-plugin/lib/src/components/GenericInput/storyUtils/layout.js

```diff
@@ -135,7 +135,7 @@ const layout = [
           key: 'pizza',
           value: 'pizza',
           metadatas: {
```

Comment:
Fwiw, `data` is its own plural, similarly for `metadata`. I didn't change `metadatas` as it would have required me to review a much larger set, so I'm just leaving a comment here on it.